getting-started/ts-example/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@opentelemetry/getting-started",
+  "version": "1.0.0",
+  "description": "This repository provides everything required to follow the OpenTelemetry Getting Started Guide",
+  "main": "app.js",
+  "scripts": {
+    "start": "ts-node app.ts"
+  },
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@opentelemetry/exporter-prometheus": "^0.3.2",
```

Comment:
Other dependencies are missing here, like `metrics, types, zipkin`, I think we should include them here. WDYT?

Reply:
Makes sense, I will add those.

Reply:
Updated.